src/eventable.js

```diff
@@ -82,11 +83,13 @@ function getEventableModule (notifyContext) {
       }
 
       const eventListeners = listeners[event]
-      if (eventListeners === undefined) return
+      if (!Array.isArray(eventListeners)) return
```

Comment:
The explicit check for `undefined` isn't really necessary, because `if (!eventListeners)` would already suffice. Instead of just shortening it down I wanted to make the code more human readable.